doc/release-notes/10626-archival-tsv.md

```diff
@@ -0,0 +1,6 @@
+Release Highlights:
+An experimental "Archival" metadatablock has been added. The purpose of the metadatablock is to enable repositories to register metadata relating to the potential archiving of the dataset at a depositor archive, whether that being your own institutional archive or an external archive i.e. a historical archive. (see #10626)
+
+To consider: 
+Add archival.tsv as release asset ? 
+Add a link ? to [Metadata Customization](https://guides.dataverse.org/en/latest/admin/metadatacustomization.html)
```

Comment:
No, I would bother creating a release asset.

Instead I would suggest that you update a future version of https://guides.dataverse.org/en/6.5/user/appendix.html following the CodeMeta example. Do you need help with this? The file to edit is doc/sphinx-guides/source/user/appendix.rst

Then we'd update this release note snippet to point to the "experimental metadata" section of the appendix.

Would we ask people to give feedback? How would you want it? In the form or issue or pull requests? As Google Group posts or Zulip messages?